docs/Seeding.md

```diff
@@ -24,6 +23,7 @@ be in your working directory with the UIPA.org source files cloned from your
 fork of the main branch of the [UIPA.org
 repository](https://github.com/CodeWithAloha/uipa).
 
+The context of the following commands should be run in the `data/seed` directory, rather than the `docs` directory.
```

Comment:
I think that it should be the top level of the working directory because that where the `manage.py` file is located.